### ISSUE TITLE
Bug: PHP line following series of line comments becomes commented out

### ISSUE THREAD

**Issue by bronius** [abrookins/WrapToColumn#38]
### Given:

```
            // Limiting to the current user so we do not subquery chatbot fields on _all_ users'
            // chatbot+ fields.
            $sub_query->condition('entity_id', $this->currentUser->id());
```

with cursor in the first line (or anywhere in the commented lines), invoke the Wrap Paragraph to Column function.

### Expected:

```
            // Limiting to the current user so we do not subquery chatbot fields
            // on _all_ users' chatbot+ fields.
            $sub_query->condition('entity_id', $this->currentUser->id());
```

### Instead:

```
            // Limiting to the current user so we do not subquery chatbot fields
            // on _all_ users' chatbot+ fields.
            // $sub_query->condition('entity_id', $this->currentUser->id());
```

**Comment by leoluk** [comment]
Duplicate of #29

**Comment by abrookins** [comment]
Should be fixed in 1.7.0, now published to the Plugins Marketplace.
